core/shared/src/test/scala-3/org/virtuslab/yaml/BaseYamlEncoderSuite.scala

```diff
@@ -125,6 +125,16 @@ class BaseYamlEncoderSuite extends munit.FunSuite:
     assertEquals(data.asYaml, expected)
   }
 
+  test("option") {
+    case class Foo(field: Option[String]) derives YamlCodec
+
+    val some = Foo(Some("some"))
+    val none = Foo(None)
+
+    assertNoDiff(some.asYaml, "field: some")
```

Comment:
Do we have a test going the other way? Or printing is not possible at all currently?

Reply:
Oh, sure it is!